src/control/server/mgmt_svc_test.go

```diff
@@ -622,3 +733,41 @@ func TestMgmtSvc_LeaderQuery(t *testing.T) {
 		})
 	}
 }
+
+func (svc *mgmtSvc) lastCall() *drpc.Call {
+	mi, _ := svc.harness.GetMSLeaderInstance()
+	return mi._drpcClient.(*mockDrpcClient).SendMsgInputCall
+}
+
+func TestMgmtSvc_PoolQuery(t *testing.T) {
```

Comment:
Going to add failure cases similar to other MgmtSvc functions? I had broken the others into separate test functions because the failure points are all pretty different.

Reply:
Yep, done.